docs/interfaces/IEnum.html

```diff
@@ -74,7 +74,7 @@ <h4>Example</h4><pre><code class="language-ts"><span class="hl-5">const</span><s
 </code><button>Copy</button></pre>
 </div><aside class="tsd-sources">
 <ul>
-<li>Defined in <a href="https://github.com/matejchalk/better-enums/blob/da32313/src/types.ts#L22">src/types.ts:22</a></li></ul></aside></li>
```

Comment:
I did not touch these files. Can’t get why they are included.

Reply:
No worries, there's a [pre-commit hook](https://github.com/matejchalk/better-enums/blob/main/.husky/pre-commit#L5-L6) which regenerates the docs :grin: